src/renderer/utils/linkSaturation.ts

```diff
@@ -0,0 +1,57 @@
+import { LinkType } from 'data/Types';
+import { ETH_BANDWIDTH_INITIAL_GBS } from 'data/constants';
+
+interface LinkSaturationMetrics {
+    linkType: LinkType;
+    totalOps: number;
+    DRAMBandwidth: number;
+    PCIBandwidth: number;
+    CLKMHz: number;
+    totalDataBytes: number;
+    initialMaxBandwidth: number;
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export const calculateLinkSaturationMetrics = ({
```

Comment:
if you are calculating X i am expoecting that X is returened.
i.e. i woudl have expected `calculateLinkSaturationMetrics` to return `LinkSaturationMetrics`, not take it as a parameter

Reply:
We talked about this before 👀
We had trouble naming this function, maybe `recalculateLinkSaturationMetrics` works better?